config/core/configmaps/features.yaml

```diff
@@ -176,3 +176,13 @@ data:
     # 1. Enabled: enabling write access for persistent volumes
     # 2. Disabled: disabling write access for persistent volumes
     kubernetes.podspec-persistent-volume-write: "disabled"
+
+    # Controls if the queue proxy podInfo is enabled or not
+    # Should be enabled when using queue proxy Options (Extensions).
```

Comment:
Can we update this to be more descriptive.

ie. Enabling this will create a mount the pod annotations in the queue proxy container at `/etc/podinfo/annotations`

And mention this is ALPHA - similar to other features

Reply:
Done